buildout.cfg

```diff
@@ -13,6 +13,10 @@ recipe = zc.recipe.egg
 eggs =
     Sphinx
     docutils
+    ZODB
+    j1m.sphinxautointerface
```

Comment:
Yes, but according to the change log, the default is hash, which is nonsensical. I'd have to set the order to something sane on every use.